section2/exercises/methods.rb

```diff
@@ -40,6 +53,10 @@ def print_name(name)
 # Look at the code you wrote for the previous YOU DO🎈 - what did you name the
 # function, and why?
 # YOUR ANSWER HERE
+# I named the function multiplication because that was what the whole purpose of the function was
```

Comment:
Instead of naming it the noun version `multiplication`, I prefer to name my functions by their purpose in verb form. So I like `multiply` better, because when I read the code, I'm thinking, "Oh, `multiply(5, 12)` must mean that the result is going to be the result of 5 times 12!" I think it just reads better.